### PR TITLE
Purge most uses of "simple" and "basic"

### PR DESCRIPTION
**Pull request by JoshKarpel**
I'm in a "better safe than slightly sorry" mood...

### CODE REVIEW

File: docs/materials/htc/part1-ex3-jobs.md [hunk 1]
```diff
@@ -9,21 +9,21 @@ HTC Exercise 1.3: Run Jobs!
 
 The goal of this exercise is to submit jobs to HTCondor and have them run on the local pool (CHTC). This is a huge step in learning to use an HTC system!
 
-**This exercise will take longer than the first two, short ones. It is the essential part of this exercise time. If you are having any problems getting the jobs to run, please ask the instructors! It is very important that you know how to run simple jobs.**
```
Comment (osg-cat) [inline docs/materials/htc/part1-ex3-jobs.md:12]: This one, I would keep as the original.

File: docs/materials/htc/part1-ex3-jobs.md [hunk 2]
```diff
@@ -9,21 +9,21 @@ HTC Exercise 1.3: Run Jobs!
...
+**This exercise will take longer than the first two, short ones. It is the essential part of this exercise time. If you are having any problems getting the jobs to run, please ask the instructors! It is very important that you know how to run jobs.**
 
-Running a Simple Job
---------------------
+Running Your First Job
+----------------------
 
 Nearly all of the time, when you want to run an HTCondor job, you first write an HTCondor submit file for it. In this section, you will run the same `hostname` command as in Exercise 1.1, but where this command will run within a job on one of the 'execute' servers in CHTC's local HTCondor pool.
 
-Here is a simple submit file for the `hostname` command:
```
Comment (osg-cat) [inline docs/materials/htc/part1-ex3-jobs.md:19]: Keep this one, too.

File: docs/materials/htc/part1-ex3-jobs.md [hunk 3]
```diff
@@ -48,15 +48,15 @@ The lines of the submit file have the following meanings:
 | `output`     | The filename where HTCondor will write the standard output from your job.                                                                                                  |
 | `error`      | The filename where HTCondor will write the standard error from your job. This particular job is not likely to have any, but it is best to include this line for every job. |
 | `log`        | The filename where HTCondor will write information about your job run. Technically not required, it is a **really** good idea to have a log file for every job.            |
-| `request_*`  | Tells HTCondor how many `cpus` and how much `memory` and `disk` we want, which is not much, because the 'hostname' executable is pretty simple                             |
+| `request_*`  | Tells HTCondor how many `cpus` and how much `memory` and `disk` we want, which is not much, because the 'hostname' executable is pretty small                             |
```
Comment (osg-cat) [inline docs/materials/htc/part1-ex3-jobs.md]: This is fine, but I would change the word “pretty” here to nothing or “very” or “relatively”.  Try defining the word “pretty” as used in this context to a non-native speaker…

File: docs/materials/htc/part1-ex3-jobs.md [hunk 4]
```diff
@@ -176,9 +176,9 @@ or perhaps a simple shell script of commands that you'd like to run within a job
         System: Linux x86_64 GNU/Linux 
         Program: ./test-script.sh
         Args: hello 42
-        ls: hostname.sub montage simple.err simple.log simple.out test-script.sh
+        ls: hostname.sub montage hostname.err hostname.log hostname.out test-script.sh
 
-    This step is **really** important! If you cannot run your executable from the command-line, HTCondor probably cannot run it on another machine, either. And debugging simple problems like this one is surprisingly difficult. So, if possible, test your `executable` and `arguments` as a command at the command-line first.
```
Comment (osg-cat) [inline docs/materials/htc/part1-ex3-jobs.md:181]: “Further, debugging …”

File: docs/materials/htc/part1-ex3-jobs.md [hunk 5]
```diff
@@ -216,7 +216,7 @@ or perhaps a simple shell script of commands that you'd like to run within a job
 !!! note
     There are Extra Challenges throughout the school curriculum. You may be better off coming back to these after you've completed all other exercises for your current working session.
 
-Below is a simple Python script that does something similar to the shell script above. Run this Python script using HTCondor.
```
Comment (osg-cat) [inline docs/materials/htc/part1-ex3-jobs.md:219]: It is simple…

File: docs/materials/htc/part2-ex1-files.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ The goal of this exercise is make input files available to your job on the execu
 Viewing a Job Sandbox
 ---------------------
 
-Before you learn to transfer files to and from your job, it is good to understand a bit more about the environment in which your job runs. When the HTCondor `starter` process prepares to run your job, it creates a new directory for your job and all of its files. We call this directory the *job sandbox*, because it is your job’s private space to play. Let’s see what is in the job sandbox for a very simple job with no special input or output files.
```
Comment (osg-cat) [inline docs/materials/htc/part2-ex1-files.md:15]: “trivial”? “minimal”?

File: docs/materials/htc/part2-ex1-files.md [hunk 7]
```diff
@@ -110,7 +110,7 @@ Transferring Output Files
 
 So far, we have relied on programs that send their output to the standard output and error streams, which HTCondor captures, saves, and returns back to the submit directory. But what if your program writes one or more files for its output? How do you tell HTCondor to bring them back?
 
-Let’s start by exploring what happens to files that a job creates in the sandbox. We will use a very simple method for creating a new file: We will copy an input file to another name.
```
Comment (osg-cat) [inline docs/materials/htc/part2-ex1-files.md:113]: I like simple here.

File: docs/materials/index.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ Lecture recording will be linked here.
 ### Intro Exercises 1: Running and Viewing Simple Jobs (Strongly Recommended)
 
 - [Exercise 1.1: Log in to the local submit machine and look around](/materials/htc/part1-ex1-login)
-- [Exercise 1.2: Experiment with basic HTCondor commands](/materials/htc/part1-ex2-commands.md)
```
Comment (osg-cat) [inline docs/materials/index.md:10]: I would leave this one.

**Review by osg-cat — CHANGES_REQUESTED** [review]
Overall, I agree with the basic/simple purge. It’s funny to see so many instances in a row like this! However, in a few cases I think the original wording is best.

**Comment by JoshKarpel** [comment]
Resolved some comments, but not others. I think the operative question should be "does using simple/basic communicate something necessary to understanding?", and I don't think it does in the remaining cases (or there is some other other, better re-wording). It should not be used to "reassure" the reader.

**Pull request merged**